app/forms/work_form.rb

```diff
@@ -96,7 +96,7 @@ def persisted?
   validates :create_date_type, inclusion: { in: %w[single range] }
 
   attribute :whats_changing, :string
-  validates :whats_changing, presence: true, if: -> { persisted? && version > 1 }
+  validates :whats_changing, presence: true
```

Comment:
The problem is that this didn't validate when the user was making a change to a deposited version 1 (since at that time the version was still 1 it bypassed the validation).

Unfortunately, this state can't be detected without having access to version status, so took a different approach to the validation by providing a dummy value when an open first version.